lore/estimators/keras.py

```diff
@@ -222,7 +223,11 @@ def build_embedding_layer(self, inputs, tower):
                     embedding = Embedding(encoder.cardinality(), embed_size, name=embed_name)
 
                 if hasattr(encoder, 'sequence_length'):
-                    embeddings[embed_name], layer = self.build_sequence_embedding(encoder, embedding, inputs, embed_name)
+                    if isinstance(encoder, Continuous):
```

Comment:
If encoders had a shape learned from their series rather than a sequence_length for Continuous, would that make this code more or less complicated?

Reply:
If I am following, it seems like the code would be identical? The need to check if it's continuous or not relates to having to pass in the reshape. Which is a bit of a quirk of Keras in my opinion - that embeddings have that extra dimension.